### DISCUSSION TITLE
Modern-Farm--How do I know if I'm planting the seeds correctly?

### DISCUSSION THREAD

**Discussion by jacksonrgoodman** [nss-day-cohort-47/C47-Discussions#19] [Help Me] [answered]
![array is array question](https://user-images.githubusercontent.com/72589713/110001945-7f933280-7cda-11eb-979d-9783d570680a.png)

**Have you searched the other help tickets?**
none yet
**List any error messages that appear**
No errors, I just don't know if the data is displaying correctly to the instructions on "Tilling the Field" of Modern Farm
**What have you googled?**
N/A, just array.isarray on mozilla.

**Describe your issue**
Hey I'm having a problem with making sure that I'm planting these seeds correctly!  I think I'm following the instructions correctly but I'm trying to make sure I followed the instructions to get the right answer instead of horribly messing up my data just to get an answer that "looks correct"

**Comment by BryanNilsen** [comment, marked as answer]
What are your tests saying in the terminal?
You can keep your tests running in one terminal window while you make changes to your code. As you make changes, the tests will run again. Read the errors and post what they're saying and if you found any resolution

**Reply by jacksonrgoodman** [reply]
My tests were coming back with errors!

when I was running the code, I accidentally put all of my seeds inside of arrays! That should only be the case for corn.js!

So I fixed the variables to no longer be an array.
(i.e. `const asparagus = {object information here}` is **correct**, _not_ `const asparagus = [{object information here]}` )

**Now I know how using `npm run test` in the terminal can help me bug test and check the project!** 
Thanks
